viewer/packages/360-images/src/annotation/Image360Annotation.ts

```diff
@@ -23,4 +23,9 @@ export interface Image360Annotation {
    * Set whether this annotation should be visible. Default: true
    */
   setVisible(visible?: boolean): void;
+
+  /**
+   * Get center of annotation, to e.g. point the camera toward it
+   */
+  getCenter(out?: Vector3): Vector3;
```

Comment:
Yes, good point.. The main reason I'm using the center is that it's easier to use out-of-the-box for the user when choosing what direction to look. I'm a little tempted to do both new :sweat_smile:

Reply:
I've decided not to do this anyway - since the position of the annotation doesn't really make sense in three dimensions, it will mostly be confusing, I think :thinking: Could have something like a Vector2 to denote size, though, but I think we can wait until the need arises